hack/update-swagger.sh

```diff
@@ -28,7 +28,7 @@ cd cmd/kubermatic-api/
 # Because kubermatic repository still contains api types, swagger generation is not consistent.
 # Sometime the  "x-go-package" point on dashboard repo and sometime on kubermatic repo.
 # To avoid this exclude kubermatic packages with this option -x k8c.io/kubermatic/v2/pkg/api/*
-#
+
 # this workaround will be removed once api will be totally removed from kubermatic repository
 run_swagger generate spec --tags=ee --scan-models -o swagger.json -x k8c.io/kubermatic/v2/pkg/api/*
```

Comment:
In order to exclude some swagger models that are located in `k8c.io/kubermatic/v2/pkg/apis` we need to tweak this pattern: 
```suggestion
run_swagger generate spec --tags=ee --scan-models -o swagger.json -x k8c.io/kubermatic/v2/pkg/api*
```
NOTE: We need to do the same for `verify-swagger.sh`.